docker/generic/Dockerfile.base

```diff
@@ -79,15 +83,70 @@ RUN echo "source /opt/ros/$ROS_DISTRO/setup.bash" >> /etc/profile.d/ros.sh && \
 #
 # Scan Autoware packages to install dependencies
 #
+# Ignore ament dependencies for now
 COPY ./ /tmp/Autoware
 RUN apt-get update && \
         apt-get install -y ros-$ROS_DISTRO-desktop-full && \
-        rosdep install -y --from-paths /tmp/Autoware/ros/src --ignore-src && \
+        rosdep install -y --from-paths /tmp/Autoware/ros/src --ignore-src --skip-keys="ament_lint_common ament_lint_auto ament_cmake_gtest ament_cmake ament_cmake_lint_cmake" && \
         rm -rf /var/lib/apt/lists/* && \
         rm -rf /tmp/Autoware  # Removed for clean environment
 
 RUN su -c "rosdep update" autoware
 
+#
+# Enable ament_cmake
+#
+
+RUN echo "deb http://repo.ros2.org/ubuntu/main xenial main" | tee /etc/apt/sources.list.d/ros2-latest.list
+RUN apt-get update
```

Comment:
Combine apt-get update and apt-get install on the same RUN command to avoid bloating the Docker layers. Also add "rm -rf /var/lib/apt/lists/*" at the end of the same RUN command.

Reply:
Yep